.travis.yml

```
@@ -32,15 +32,5 @@ deploy:
     skip_cleanup: true
     local-dir: "pkgs/releases"
     on:
-      branch: "debian-enterprise"
-      condition: -d "pkgs/releases"
-  - provider: packagecloud
-    username: "citusdata"
-    repository: "$PKG_REPOTYPE-nightlies"
-    token: $PACKAGECLOUD_API_TOKEN
-    dist: $TARGET_PLATFORM
-    skip_cleanup: true
-    local-dir: "pkgs/nightlies"
-    on:
-      branch: "debian-enterprise"
-      condition: -d "pkgs/nightlies"
+      branch: "debian-enterprise-8.3"
+      condition: -d "pkgs/releases"
```

```suggestion
      condition: -d "pkgs/releases"
      
```

According to POSIX, every text file should end with a `\n`

---

See https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206 for more details on the standard